kostyor/rest_api.py

```diff
@@ -8,21 +8,16 @@
 from keystoneauth1.identity import v2
 import six
 
+from kostyor import conf
 from kostyor.common import constants
 from kostyor.inventory import discover
 from kostyor.inventory import upgrades
 
 from kostyor.db import api as db_api
-from kostyor.db.api import db_session
 
 app = Flask(__name__)
 
 
-@app.teardown_appcontext
-def shutdown_session(exception=None):
```

Comment:
Actually, it's a good thing and I think we need to move toward:

* `commit()` stuff here if there's no exception
* `rollback()` stuff here if there's an exception

In that case we will have a clear transaction scope - one http request. It's easy to track, it's easy to debug.. and we don't need to remove some stuff if something wrong was committed.